dace/sdfg/validation.py

```diff
@@ -155,7 +155,7 @@ def validate_state(state: 'dace.sdfg.SDFGState',
             raise
         except Exception as ex:
             raise InvalidSDFGNodeError("Node validation failed: " + str(ex),
-                                       sdfg, state_id, nid)
+                                       sdfg, state_id, nid) from ex
```

Comment:
@orausch why is this here?

Reply:
This was helpful for me while debugging so I thought it could be useful for master. I can remove it if you disagree